dags/common/cleanup.py

```diff
@@ -1,9 +1,13 @@
+import re
+
 from bleach import clean
 from bleach.html5lib_shim import Filter
 from bleach.sanitizer import Cleaner
 
 
 def clean_whitespace_characters(input):
+    if type(input) == list:
```

Comment:
I prefer to use `isinstance()`.
Also, is the list/string the only input possibility? Maybe you can add type annotation?